packages/acceptance-tests/pkg-tests-specs/sources/commands/exec.test.ts

```diff
@@ -28,7 +28,7 @@ describe(`Commands`, () => {
       `it should inject binaries the workspace has access to`,
       makeTemporaryEnv({}, async ({path, run, source}) => {
         await xfs.mkdirPromise(ppath.join(path, `bin` as Filename));
```

Comment:
Is something preventing this and other `as Filename`s from getting removed?

Reply:
Hm no; I used a regexp to blanket-update the repo, it may have missed some patterns